addons/account/models/chart_template.py

```diff
@@ -1029,7 +1031,33 @@ def _get_account_reconcile_model(self, template_code):
                 "match_same_currency": True,
                 "allow_payment_tolerance": False,
                 "match_partner": True,
-            }
+            },
+            "reconcile_invoice": {
```

Comment:
@hupo-odoo Thanks for the quick review!
This is done now :) Also added another commit in this PR, removing the Bank Fees. Talking to CHKL, he said it could be done in the same PR, but if you think it's better doing it in another, let me know!